### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by fsimkovic**
ignore the first commit, it's simply code formatting using [Black](https://pypi.org/project/black/). The rest are some good improvements to the code I think.

### CODE REVIEW

**Comment by fsimkovic** [comment]
please run all your tests, unittests pass but if you have any others it'd be good to get them checked

**Comment by fsimkovic** [comment]
btw feel free to merge once you're happy. also good to delete the branch afterwards

**Comment by fsimkovic** [comment]
Pushed an update, please re-test. Should be fixed now

File: simbad/util/pdb_util.py [hunk 1]
```diff
@@ -22,29 +22,34 @@ def __init__(self):
...
+        struct = cls()
         if input_file.endswith(".dat"):
-            pdb_str = read_dat(input_file)
-            self.pdb_input = iotbx.pdb.input(source_info=None, lines=pdb_str)
+            struct.pdb_input = iotbx.pdb.input(source_info=None, lines=read_dat(input_file))
         elif input_file.endswith(".pdb") or input_file.endswith(".ent"):
-            self.pdb_input = iotbx.pdb.pdb_input(file_name=input_file)
+            struct.pdb_input = iotbx.pdb.pdb_input(file_name=input_file)
         elif input_file.endswith(".ent.gz"):
             with gzip.open(input_file, "rb") as f_in:
```
Comment (hlasimpk) [inline simbad/util/pdb_util.py]: I don't think this line is needed and the line below should be indented once more

File: simbad/mr/__init__.py [hunk 2]
```diff
@@ -389,14 +386,11 @@ def generate_script(self, result):
         diff_mapout1 = os.path.join(ref_workdir, "{0}_refmac_2fofcwt.map".format(result.pdb_code))
         diff_mapout2 = os.path.join(ref_workdir, "{0}_refmac_fofcwt.map".format(result.pdb_code))
 
-        if isinstance(result, AmoreRotationScore) or isinstance(result, PhaserRotationScore):
-            pdb_struct = PdbStructure()
-            pdb_struct.from_file(result.dat_path)
+        if isinstance(result, (AmoreRotationScore, PhaserRotationScore)):
+            pdb_struct = PdbStructure.from_file(result.dat_path)
             mr_pdbin = os.path.join(self.output_dir, result.pdb_code + ".pdb")
-            pdb_struct.save(mr_pdbin)
```
Comment (hlasimpk) [inline simbad/mr/__init__.py]: pdb_struct.save got deleted

**Comment by hlasimpk** [comment]
There were a couple of issues but I've fixed those :)

**Pull request merged**